src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/bulkscan/validation/out/OcrValidationResult.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.validation.out;
+
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.apache.commons.collections.CollectionUtils.isNotEmpty;
+
+@Getter
+public class OcrValidationResult {
```

Comment:
->lib

Reply:
Moved into the BSP Commons Lib as requested